packages/airnode-deployer/src/cli/index.ts

```diff
@@ -51,7 +51,6 @@ const cliExamples = [
   'remove-with-deployment-details --airnode-address 0x6abEdc0A4d1A79eD62160396456c95C5607369D3 --stage dev --cloud-provider aws --region us-east-1',
 ];
 
-drawHeader();
```

Comment:
I am not so sure about this change. I'd prefer the header everywhere for consistency (but I know this is subjective).